code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -92,6 +100,10 @@
 	environment_type = "basalt"
 	floor_variance = 15
 	digResult = /obj/item/stack/ore/glass/basalt
+	/var/list/fissure_gas_mix = list("plasma=42;n2=32;TEMP=600" = 3,
```

Comment:
remove the first `/`

Reply:
removed the slash, does it also fix the var not existing above?

Reply:
nope now it doesn't compile. welp, debugging and readings docs, fix shortly

Reply:
Well it's fixed but I really didn't want to modify the base asteroid tile. I hadn't realized the dig behavior was declared at the asteroid level. I think I need to approach this differently, I had a worst grasp of this than I thought. Not sure how to proceed, I should probably retract this while I hammer out a better solution. I guess I leave it to your collective wisdom - does this work, or should I start over and do it properly and put everything under basalt?

Reply:
actually - thinking about it again- finding gas vents on asteroids could also be pretty interesting.  I'm satisfied with this, Was there anything else you'd like me to review ShizCalev? It should work as expected now